### PR TITLE
Add active projects section

### PR DESCRIPTION
**Pull request by MohammadPCh**

### CODE REVIEW

File: src/components/views/userProfile/donationsTab/recurringTab/FilterMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+					<B>{formatMessage({ id: 'label.tokens' })}</B>
+					{config.OPTIMISM_CONFIG.SUPER_FLUID_TOKENS.map(token => (
+						<FeatureItem key={token.id}>
+							<CheckBox
+								label={token.name}
+								onChange={e => {
+									handleSelectFilter(e, token);
+								}}
+								checked={
+									false
```
Comment (jainkrati) [inline src/components/views/userProfile/donationsTab/recurringTab/FilterMenu.tsx:74]: please confirm if these needs to be dynamic instead of false always

**Review by jainkrati — APPROVED** [review]

**Comment by jainkrati** [comment]
please merge latest branch and resolve commits and then we are good to merge.

**Comment by MohammadPCh** [comment]
@jainkrati Sorry, It was my fault, this branch is not ready :(

File: src/components/views/userProfile/donationsTab/recurringTab/FilterMenu.tsx (hunk 1 above)
Comment (MohammadPCh) [reply src/components/views/userProfile/donationsTab/recurringTab/FilterMenu.tsx:74]: @jainkrati I will handle it when backend is ready

**Pull request merged**